registries/_format/binary.md

```diff
@@ -12,10 +14,86 @@ layout: default
 
 Base type: `{{ page.base_type }}`.
 
-The `{{page.slug}}` format represents any sequece of octets. This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
+The `{{page.slug}}` format represents any sequence of octets. This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
+
+### Upgrading from < 3.1 to >= 3.1
+
+#### Entire response body
+
+```yaml
+#OAS 3.0
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            image/png:
+              schema:
+                type: string
+                format: binary
+
+#OAS 3.1
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            image/png:
+              schema: true #optional, some linters require a schema property
+```
+
+#### Encoded property as part of a structured response
+
+Using the binary format as part of a structured response for a property or an array is invalid. The only exception is for `multipart/form-data` content.
```

Comment:
Is "structured response" common usage to mean JSON?  Does it typically include things like `multipart/form-data`?  I would not think of `application/json` and `multipart/form-data` as the same sort of thing, but I spend too much time using RFCs.

Reply:
JSON or XML or YAML, etc... anything that can be used as a serialization format more or less.

Reply:
(I'd mark this resolved but apparently I can't)

Reply:
resolved